src/containers/home.js

```diff
@@ -8,25 +8,25 @@ export class home extends Component {
     return (
       <Container >
         <Row className="boot-home-logo">
-        <img className="img-fluid" src={icebreakersv8} alt="icebreakers logo"/>
+          <img className="img-fluid" src={icebreakersv8} alt="icebreakers logo"/>
         </Row>
         <Row >
-        <Col className="col-2"></Col>
-        <Col className="col align-self-center">
-          <Link to="/create_room">
-            <button className="homeCreateBtn" ><h3 className="homeCreateBtnText">Create a Room</h3></button>
-          </Link>
-        </Col>
-        <Col className="col-2"></Col>
+          <Col className="col-2"></Col>
+          <Col className="col align-self-center">
+            <Link to="/create_room">
+              <button className="homeCreateBtn"><span className="homeCreateBtnText">Create a Room</span></button>
+            </Link>
+          </Col>
+          <Col className="col-2"></Col>
         </Row>
         <Row>
-        <Col className="col-2"></Col>
-        <Col className="col align-self-center">
-          <Link to="/login">
-            <button className="homeJoinBtn"><h3 className="homeJoinBtnText">Join a Room</h3></button>
-          </Link>
-          </Col>
-        <Col className="col-2"></Col>
+          <Col className="col-2"></Col>
+          <Col className="col align-self-center">
+            <Link to="/login">
+              <button className="homeJoinBtn"><span className="homeJoinBtnText">Join a Room</span></button>
+            </Link>
+            </Col>
+          <Col className="col-2"></Col>
```

Comment:
Indentation is important for making the code more scanable by humans. You can use a linter to enforce stuff like this for you, too.